Minimal_test/Test.html

```diff
@@ -0,0 +1,71 @@
+<html>
+  <head>
+	  <title>Santiagos Webseite</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mocha/3.2.0/mocha.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mocha/3.2.0/mocha.js"></script>
+    <script>
+      mocha.setup('bdd'); // minimal setup
+    </script>
+    <!-- add chai -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/chai/3.5.0/chai.js"></script>
+    <script>
+      // chai has a lot of stuff, let's make assert global
+      let assert = chai.assert;
+    </script>
+    
+  </head>
+  <body>
+    <script>
+      function parseVcard(vCard) {
+        let user = new Object();
+        
+        user = {
+          secondName: vCard.slice(vCard.search(" N:")+3, vCard.search(";")),
+          firstName: vCard.slice(vCard.search(";")+1, vCard.search(";;;")),
+          birthDay: (vCard.slice(vCard.search("BDAY:")+5,vCard.search("EMAIL")-1).slice(4,6) + "." + vCard.slice(vCard.search("BDAY:")+5,vCard.search("EMAIL")-1).slice(6)) + "." + vCard.slice(vCard.search("BDAY:")+5,vCard.search("EMAIL")-1).slice(0,4),
+          email: vCard.slice(vCard.search("EMAIL")).slice(vCard.slice(vCard.search("EMAIL")).search(":") +1, vCard.slice(vCard.search("EMAIL")).search("TEL;") -1),
+          phone: vCard.slice(vCard.search("TEL;")).slice(vCard.slice(vCard.search("TEL;")).search(":")+1,vCard.slice(vCard.search("TEL;")).search("ADR")-1),
+          street: vCard.slice(vCard.search(":;;")+3).slice(0,vCard.slice(vCard.search(":;;")+3).search(";")),
+          city: vCard.slice(vCard.search(":;;")+3).slice(vCard.slice(vCard.search(":;;")+3).search(";")+1,vCard.slice(vCard.search(":;;")+3).search(";;")),
+          zipCode: vCard.slice(vCard.search(":;;")+3).slice(vCard.slice(vCard.search(":;;")+3).search(";;")+2,vCard.slice(vCard.search(":;;")+3).search("; REV"))
+        }
+        return user;
+      }
+    </script>
+
+    <script>
+      const vcard = "Last Match: BEGIN:VCARD VERSION:4.0 N:Musterperson;Alex;;; FN:Alex Musterperson BDAY:20022002 EMAIL;TYPE=home:muster@person.alex TEL;TYPE='cell,home':055544332211 ADR;TYPE=home:;;Hauptstr. 23;Plauen;;42230; REV:20220110T082823Z END:VCARD"
+
+      describe("parseVcard", function() {
+        it("parses the second name correctly", function() {
+          assert.equal(parseVcard(vcard).secondName, "Musterperson");
+        });
+        it("parses the first name correctly", function() {
+          assert.equal(parseVcard(vcard).firstName, "Alex")
+        });
+        it("parses the birthDay correctly", function(){
+          assert.equal(parseVcard(vcard).birthDay,"20.02.2002")
+        });
```

Comment:
sorry: das war mein Fehler, weil ich dir ein doofes Datum gegeben habe (bei `20022002` ist nicht ersichtlich, was Jahr und was Monat ist). Setz mal bitte in den Testdaten `BDAY:﻿19870605`  und teste auf `05.06.1987` ab ... da sind Monat und Tag vertauscht.
Liegt aber, wie gesagt, daran, dass die Testdaten doof gewählt waren :)